rpmbuild/copr-rpmbuild.spec

```diff
@@ -51,6 +51,8 @@ BuildRequires: python3-backoff >= 1.9.0
 
 %if 0%{?fedora} || 0%{?rhel} > 7
 BuildRequires: argparse-manpage
+%elif 0%{?openEuler}
+BuildRequires: python3-argparse-manpage
```

Comment:
Can we BuildRequire `/usr/bin/argparse-manpage`?

Reply:
Good idea!